src/models/utils/index.ts

```diff
@@ -19,3 +24,23 @@ export function onlyHasFields(obj: object, fields: Array<string>): boolean {
 export function isFlagEnabled(Flags: number, checkFlag: number): boolean {
     return (checkFlag & Flags) === checkFlag
 }
+
+/**
+ * Hashes the Transaction object as the ledger does. Only valid for signed Transactions.
+ * 
+ * @param {Transaction} tx A transaction to hash. Tx must be signed.
+ * @returns {string} A hash of tx
+ * @throws {ValidationError} if the Transaction is unsigned
+ */
+
+// TODO: Investigate 0x53545800 if single-signing, or 0x534D5400 if multisigning - https://xrpl.org/serialization.html#blob-fields 
+// Could be a bug in the xrpl-py method!
```

Comment:
This is not a bug, that's a signing-specific prefix and not relevant for already-signed transactions.

Reply:
Handled in #1534 which moves the hashes functions to the utils folder